<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegexpFilterEditor</class>
 <widget class="QDialog" name="RegexpFilterEditor">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>116</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Edit regexp</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QComboBox" name="InclusionComboBox">
       <item>
        <property name="text">
         <string>Include</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Exclude</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="EntryTypeComboBox">
       <item>
        <property name="text">
         <string>files</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>directories</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>entries</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>by regexps</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="placeholderText">
        <string>Example: invoice.*\.pdf;.manuscript.+</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;http://www.cplusplus.com/reference/regex/ECMAScript/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;See reference&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RegexpFilterEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RegexpFilterEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>RegexpFilterEditor</receiver>
   <slot>ValidateForm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ValidateForm()</slot>
 </slots>
</ui>
